test_cc_functionality.py

```
@@ -0,0 +1,147 @@
+        result = send_email(
+            to="primary@example.com",
+            subject="Test CC Functionality",
+            body="This is a test email with CC recipients.",
+            cc="cc1@example.com, cc2@example.com",
+            bcc="bcc@example.com"
+        )
+        print(f"Expected error result: {result}")
+        assert "No email backend configured" in result
+        print("✅ Mock email sending test passed!")
+        
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)